about/policy/sgid/index.md

```diff
@@ -29,41 +46,52 @@ A corresponding record to the [Data Stewardship spreadsheet](https://docs.google
+Dates are acceptable in table names when content belongs in either the `census`, `political`, or `tax` ISO categories to identify datasets where the most current data may not reflect the current calendar year. For all other categories, dates should only be used for tables that are no longer effective/active or soon-to-be effective/active. This implies that the **most current and relevant** data contains **no date suffix**. Historical or future data _will_ contain a date suffix to help identification at a glance. The best way to learn about a dataset is to view the data page metadata.
 
-## Updating and Editing Existing Tables
+## Updating and Editing Existing Datasets
 {: .text-left }
 
 No edits will be made to the internal SGID from 7:00PM to 10:00AM MST to allow for automated scripts to push accurate updates to ArcGIS Online and ArcGIS Server. AGRC will ensure that data is production-ready before the editing windows ends.
 
-All database updates will be done during the editing window using [swapper](https://github.com/agrc/swapper) or some other automated process to reduce the amount of time with incomplete data. 
+All database updates will be done during the editing window using [Swapper](https://github.com/agrc/swapper) or some other automated process to reduce the amount of time with incomplete data.
 
 All data updates being pushed into the SGID will have removed empty geometries and duplicate records with [Sweeper](https://github.com/agrc/sweeper) or another automated process.
 
-All edits made to the internal SGID will be captured in `Last update date` in the metadata, in the [SGID Update log](https://docs.google.com/spreadsheets/d/11ASS7LnxgpnD0jN4utzklREgMf1pcvYjcXcIcESHweQ/edit#gid=0), and as an update on the AGRC website data page.
+All edits made to the internal SGID will be captured in the `Last update date` in the metadata, in the [SGID Update log](https://docs.google.com/spreadsheets/d/11ASS7LnxgpnD0jN4utzklREgMf1pcvYjcXcIcESHweQ/edit#gid=0), and as an update on the AGRC website data page.
 
 If making a schema change, please consider updating any domain descriptions that do not match the domain value. See this [document](https://docs.google.com/document/d/12bdtmtv-ZVF9g-lFshbobx8DbKX9LfbL44HawLjFTVw/edit?usp=sharing) to learn more about this effort.
 
-## Removing SGID Products
+## Removing SGID Datasets
 {: .text-left }
 
 When deprecating data or services in the SGID, please follow the steps outlined in the [Porter](https://github.com/agrc/porter) project.
```

Comment:
**thought**: maybe this isn't necessary any longer with the other review suggestions?

Reply:
I see what you're saying about Porter and I agree. I guess I'm still approaching this whole page with the mindset of "this is our policy for adding datasets to the internal SGID database." Thus, while Porter definitely handles a range of products, in the context of the page "datasets" makes more sense.